src/components/Composer/index.tsx

```diff
@@ -140,40 +55,50 @@ const getNextChars = (str, cursorPos) => {
 
 // Enable Markdown parsing.
 // On web we like to have the Text Input field always focused so the user can easily type a new chat
-function Composer({
-    value,
-    defaultValue,
-    maxLines,
-    onKeyPress,
-    style,
-    shouldClear,
-    autoFocus,
-    translate,
-    isFullComposerAvailable,
-    shouldCalculateCaretPosition,
-    numberOfLines: numberOfLinesProp,
-    isDisabled,
-    forwardedRef,
-    navigation,
-    onClear,
-    onPasteFile,
-    onSelectionChange,
-    onNumberOfLinesChange,
-    setIsFullComposerAvailable,
-    checkComposerVisibility,
-    selection: selectionProp,
-    isReportActionCompose,
-    isComposerFullSize,
-    ...props
-}) {
+function Composer(
+    {
+        value,
+        defaultValue,
+        maxLines = -1,
+        onKeyPress = () => {},
+        style,
+        shouldClear = false,
+        autoFocus = false,
+        isFullComposerAvailable = false,
+        shouldCalculateCaretPosition = false,
+        numberOfLines: numberOfLinesProp = 0,
+        isDisabled = false,
+        onClear = () => {},
+        onPasteFile = () => {},
+        onSelectionChange = () => {},
+        onNumberOfLinesChange = () => {},
+        setIsFullComposerAvailable = () => {},
+        checkComposerVisibility = () => false,
+        selection: selectionProp = {
+            start: 0,
+            end: 0,
+        },
+        isReportActionCompose = false,
+        isComposerFullSize = false,
+        ...props
+    }: ComposerProps,
+    ref: ForwardedRef<React.Component<AnimatedProps<TextInputProps>>>,
+) {
     const theme = useTheme();
     const styles = useThemeStyles();
     const {windowWidth} = useWindowDimensions();
-    const textRef = useRef(null);
-    const textInput = useRef(null);
-    const initialValue = defaultValue ? `${defaultValue}` : `${value || ''}`;
+    const navigation = useNavigation();
+    const textRef = useRef<HTMLElement & RNText>(null);
+    const textInput = useRef<HTMLTextAreaElement & TextInput>();
```

Comment:
```suggestion
    const textInput = useRef<HTMLTextAreaElement & TextInput>(null);
```

Let's bring back the `null`.